path.go

```diff
@@ -0,0 +1,69 @@
+package redisgraph
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Path struct {
+	Nodes []interface{}
+	Edges []interface{}
+}
+
+func PathNew(nodes []interface{}, edges []interface{}) Path {
```

Comment:
`func PathNew(nodes []Node, edges []Edge) Path {`

Reply:
the constructor will convert from array of interface to array of node/edge